### DISCUSSION TITLE
Welcome to rwb Discussions!

### DISCUSSION THREAD

**Discussion by john-noss-m2gen** [AsterInsights/rwb#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Michelle-Churchman** [comment]
Thanks, John! I think this is a great place to thought share for ORIEN/Aster Insights data users!

**Comment by tyrellis88** [comment]
@john-noss-m2gen @Michelle-Churchman @owynrichen

Please see the attached code that Ken Shain's team made to allow for Azure queries by SLID. In this instance, the SDR was used as the source data to check the data on Azure. The process could likely be modified to support IMPs on the member side.

I will let Dr. Noss explain everything else.

[SourceShare.zip](https://github.com/user-attachments/files/17031983/SourceShare.zip)
